docs/server-versions-dependencies.md

```diff
@@ -0,0 +1,167 @@
+---
+id: server-versions-and-dependencies
+title: Versions and dependencies
+sidebar_label: Versions & dependencies
+---
+
+## Overview
+
+This page details some of the version specific requirements and dependencies needed to build and run an instance of Temporal.
+
+All versions of Temporal require that [Go v1.15+](https://golang.org/dl/) is installed in the environment.
+
+## Dependencies
+
+Temporal offers official support for and is tested against dependencies with the exact versions described below.
+
+### Go Packages
+
+| Temporal version | Go packages |
+|------------------|-------------|
+| [1.3.2](https://github.com/temporalio/temporal/tree/v1.3.2) | [go.mod](https://github.com/temporalio/temporal/blob/v1.3.2/go.mod) |
+| [1.3.1](https://github.com/temporalio/temporal/tree/v1.3.1) | [go.mod](https://github.com/temporalio/temporal/blob/v1.3.1/go.mod) |
+| [1.3.0](https://github.com/temporalio/temporal/tree/v1.3.0) | [go.mod](https://github.com/temporalio/temporal/blob/v1.3.0/go.mod) |
+| [1.2.2](https://github.com/temporalio/temporal/tree/v1.2.2) | [go.mod](https://github.com/temporalio/temporal/blob/v1.2.2/go.mod) |
+| [1.2.1](https://github.com/temporalio/temporal/tree/v1.2.1) | [go.mod](https://github.com/temporalio/temporal/blob/v1.2.1/go.mod) |
+| [1.1.1](https://github.com/temporalio/temporal/tree/v1.1.1) | [go.mod](https://github.com/temporalio/temporal/blob/v1.1.1/go.mod) |
+| [1.1.0](https://github.com/temporalio/temporal/tree/v1.1.0) | [go.mod](https://github.com/temporalio/temporal/blob/v1.1.0/go.mod) |
+| [1.0.0](https://github.com/temporalio/temporal/tree/v1.0.0) | [go.mod](https://github.com/temporalio/temporal/blob/v1.0.0/go.mod) |
+
+### Persistence
+
+The only required dependency is a database, and there are multiple types of databases that are supported.
+
+- **Cassandra v3.11**
+- **MySQL v5.7**
+- **PostgreSQL v9.6** (supported since Temporal v1.2.1)
+
+### Visibility via search
```

Comment:
```suggestion
### Visibility via search (optional)
```

is this the best name for this capability? we call it "workflow search" below too. can we standardize?